[2016-02-17 15:21:54 -0500] [42282] [INFO] Starting gunicorn 19.1.1
[2016-02-17 15:21:54 -0500] [42282] [ERROR] Connection in use: ('0.0.0.0', 8000)
[2016-02-17 15:21:54 -0500] [42282] [ERROR] Retrying in 1 second.
[2016-02-17 15:21:55 -0500] [42282] [ERROR] Connection in use: ('0.0.0.0', 8000)
[2016-02-17 15:21:55 -0500] [42282] [ERROR] Retrying in 1 second.
[2016-02-17 15:21:56 -0500] [42282] [ERROR] Connection in use: ('0.0.0.0', 8000)
[2016-02-17 15:21:56 -0500] [42282] [ERROR] Retrying in 1 second.
[2016-02-17 15:21:57 -0500] [42282] [ERROR] Connection in use: ('0.0.0.0', 8000)
[2016-02-17 15:21:57 -0500] [42282] [ERROR] Retrying in 1 second.
[2016-02-17 15:21:58 -0500] [42282] [ERROR] Connection in use: ('0.0.0.0', 8000)
[2016-02-17 15:21:58 -0500] [42282] [ERROR] Retrying in 1 second.
[2016-02-17 15:21:59 -0500] [42282] [ERROR] Can't connect to ('0.0.0.0', 8000)
